projects/gstreamer/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://gstreamer.freedesktop.org/"
+primary_contact: "bilboed@bilboed.com"
+sanitizers:
+  - address
+  - memory
```

Comment:
Assuming you've verified that msan works and doesn't give false positives right away.

Reply:
Should I just leave the default then ? (i.e. nothing)